e2e/interchaintestv8/ibc_eureka_test.go

```diff
@@ -230,17 +230,26 @@ func (s *IbcEurekaTestSuite) SetupSuite(ctx context.Context, proofType operator.
 			beaconAPI = eth.BeaconAPIClient.GetBeaconAPIURL()
 		}
 
+		var sp1Config string
+		switch prover {
+		case testvalues.EnvValueSp1Prover_Mock:
+			sp1Config = testvalues.EnvValueSp1Prover_Mock
+		case testvalues.EnvValueSp1Prover_Network:
+			sp1Config = "env"
```

Comment:
Could probably be a constant?

Reply:
I think in the future, if we ever need to use the other enum variants, I can define each enum variant more robustly through an interface. Until then, I think we can leave it as is?

Reply:
I guess. It's not a big deal, just a bit inconsistent with the rest of the suite.

Reply:
If you don't want to change it, you can merge this whenever